r-pkg/R/es_search.R

```diff
@@ -277,7 +276,8 @@ es_search <- function(es_host
 
     # Find a safe path to write to and create it
     repeat {
-        out_path <- file.path(intermediates_dir, uuid::UUIDgenerate())
+        random_dirname <- sprintf("tmp-%s", .random_string(36L))
+        out_path <- file.path(intermediates_dir, random_dirname)
         if (!dir.exists(out_path)) {
```

Comment:
Haha I was gonna write something about "what if someone had set a seed and not cleaned up old directories," since now `.random_string()` gives reproducible results if the seed was set, but I forgot how we protect against collisions with `repeat` and `break` 😂

Reply:
Ha yes I DISTINCTLY remember when we talked about whether this was overkill when we first added it for paths 😂 

Excellent point about this being affected by random seed, I hadn't thought about that.

Reply:
Maybe it's worth a little comment in the helper func but other than that I think this is fine to ship!

Reply:
sure! Added a little more description in https://github.com/uptake/uptasticsearch/pull/240/commits/8327487a3d803b0bf7224dd08aeac7382cd2db59. I'll merge this once CI passes.